config/api.yml

```diff
@@ -477,10 +477,21 @@
       :post:
       - :name: query
         :identifier: cloud_volume_show_list
+      - :name: delete
+        :identifier: cloud_volume_delete
+      :delete:
```

Comment:
we don't expose DELETE method on the collection itself, so this is not consistent with the others.  DELETE targets should be fully identified by the URL itself, even though the standard does not prohibit it, some other vendors just ignore the body.  Let's stick to POST for bulk delete unless this is an issue.  Thanks.